content/pt-br/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -0,0 +1,428 @@
+---
+reviewers:
+- bprashanth
+- liggitt
+- thockin
```

Comment:
This section is not used in the Portuguese localization. It still needs to be removed.

Reply:
Hey @DonatoHorn! Please check this suggestion. After this, we can approve :D